web/src/main/kotlin/no/nav/su/se/bakover/web/AuditLogger.kt

```diff
@@ -0,0 +1,159 @@
+package no.nav.su.se.bakover.web
+
+import no.nav.su.se.bakover.common.auditLogg
+import no.nav.su.se.bakover.domain.Fnr
+import java.lang.String.join
+import java.util.UUID
+
+data class AuditLogEvent(
+    val navIdent: String,
+    val berørtBrukerId: Fnr,
+    val action: Action,
+    val behandlingId: UUID?,
+    val callId: String?,
+) {
+    /**
+     * Hva slags CRUD-operasjon blir gjort
+     */
+    enum class Action(val value: String) {
+        /** Bruker har sett data. */
+        ACCESS("audit:access"),
+
+        /** Bruker har endret data */
+        UPDATE("audit:update"),
+
+        /** Bruker har lagt inn nye data */
+        CREATE("audit:create"),
+
+        /** Minimalt innsyn, f.eks. ved visning i liste. */
+        SEARCH("audit:search"),
+    }
+}
+
+enum class CefFieldName(val kode: String) {
+    /**
+     * Tidspunkt for når hendelsen skjedde.
+     */
+    EVENT_TIME("end"),
+
+    /**
+     * Brukeren som startet hendelsen (saksbehandler/veileder/...).
+     */
+    USER_ID("suid"),
+
+    /**
+     * Bruker (søker/part/...) som har personopplysninger som blir berørt.
+     */
+    BERORT_BRUKER_ID("duid"),
+
+    /**
+     * Reservert til bruk for "Behandling". Det er godkjent med både behandlingsUuid
+     * og behandlingsId, men førstnevnte er foretrukket. Denne skal unikt identifisere
+     * behandlingen.
+     *
+     * @see CefFields.forBehandling
+     */
+    BEHANDLING_VERDI("flexString2"),
+
+    /**
+     * Reservert til bruk for "behandlingId".
+     *
+     * @see CefFields.forBehandling
+     */
+    BEHANDLING_LABEL("flexString2Label"),
+
+    /**
+     * Call-id, prosess-id
+     */
+    CALL_ID("sproc"),
+
+    /**
+     * Om handlingen blir tillatt eller ikke (permit/deny)
+     */
+    DECISION_VERDI("flexString1"),
+    DECISION_LABEL("flexString1Label");
+}
+
+data class CefField(val cefFieldName: CefFieldName, val value: String)
+
+/**
+ * Logger til auditlogg på formatet
+ *
+ * CEF:0|su-se-bakover|auditLog|1.0|audit:access|su-se-bakover audit log|INFO|
+ * end=1618308696856 suid=X123456 duid=90147898270
```

Comment:
```suggestion
 * end=1618308696856 suid=X123456 duid=01010199999
```

Bare i tilfelle noen har det fødselsnummeret og googler seg selv.